internal/install/execution/go_task_recipe_executor.go

```diff
@@ -163,11 +152,61 @@ func (re *GoTaskRecipeExecutor) Execute(ctx context.Context, r types.OpenInstall
 		return goTaskError
 	}
 
-	re.setOutput(outputFile.Name())
+	// remove output file immediately since install did not error
+	// will be nil for recipes other than infrastructure-agent-installer
+	if nil != cliOutputFile {
+		os.Remove(cliOutputFile.Name())
+	}
+	re.setOutput(outputJSONFile.Name())
 
 	return nil
 }
 
+func configureRecipeOutputCapture(re *GoTaskRecipeExecutor, recipeName string, recipeVars types.RecipeVars, cliOutputFile *os.File) (*LineCaptureBuffer, *LineCaptureBuffer, *os.File, error) {
+	silentInstall, _ := strconv.ParseBool(recipeVars["assumeYes"])
+	if silentInstall {
+		return NewLineCaptureBuffer(&bytes.Buffer{}), NewLineCaptureBuffer(&bytes.Buffer{}), nil, nil
+	}
+
+	// Create a temporary cli output file only for non-silent infra installs
+	if recipeName == "infrastructure-agent-installer" {
```

Comment:
@mbruzina I wasn't expecting this hard-wiring on infra. 
Could we expose the functionality to ship the output to NR1 log from a flag pass through the NR_CLI_OUTPUT json

Reply:
@Julien4218 sure.